### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by diederikdehaas**
Fix various spelling errors as reported by Debian's lintian tool.

Condtional -> Conditional
Interal -> Internal
adderted -> asserted
cummulative -> cumulative
casued -> caused
exlusive -> exclusive
occurance -> occurrence
occured -> occurred
occurences -> occurrences
onthe -> on the
preceeding -> preceding
reponses -> responses
satisifed -> satisfied
successfuly -> successfully
transfered -> transferred
transfering -> transferring

Link: https://lore.kernel.org/lkml/20230121184648.38487-1-didi.debian@cknow.org/
Signed-off-by: Diederik de Haas <didi.debian@cknow.org>

### CODE REVIEW

File: KNL/events/KnightsLanding_core.json
Comment (edwarddavidbaker) [inline KNL/events/KnightsLanding_core.json:894]: Looks good to me. Please increase the version in the header at the top of this file.

File: KNL/events/KnightsLanding_matrix.json
Comment (edwarddavidbaker) [inline KNL/events/KnightsLanding_matrix.json:274]: Looks good to me. Please increase the version in the header at the top of this file.

File: KNL/events/KnightsLanding_uncore.json
Comment (edwarddavidbaker) [inline KNL/events/KnightsLanding_uncore.json:1335]: Looks good to me. Please increase the version in the header at the top of this file.

File: KNM/events/KnightsLanding_core.json
Comment (edwarddavidbaker) [inline KNM/events/KnightsLanding_core.json:894]: Looks good to me. Please increase the version in the header at the top of this file.

File: KNM/events/KnightsLanding_matrix.json
Comment (edwarddavidbaker) [inline KNM/events/KnightsLanding_matrix.json:274]: Looks good to me. Please increase the version in the header at the top of this file.

File: KNM/events/KnightsLanding_uncore.json
Comment (edwarddavidbaker) [inline KNM/events/KnightsLanding_uncore.json:1335]: Looks good to me. Please increase the version in the header at the top of this file.

File: SLM/events/Silvermont_core.json
Comment (edwarddavidbaker) [inline SLM/events/Silvermont_core.json:908]: Looks good to me. Please increase the version in the header at the top of this file.

File: WSM-EP-DP/events/WestmereEP-DP_core.json
Comment (edwarddavidbaker) [inline WSM-EP-DP/events/WestmereEP-DP_core.json:881]: Looks good to me. Please increase the version in the header at the top of this file.

**Review by edwarddavidbaker — CHANGES_REQUESTED** [review]
@diederikdehaas Thank you for creating the pull request. I'll also add the spelling checks to our release tooling. Event list files are generated from an Intel database. The overall flow is roughly the following.
```
Intel database -> Intel repo -> This repo
```

* Broadwell and Cascade Lake platforms need to be generated from the database.
* TMA files also need to be updated internally and then synced to this repo.
* JKT, KNL, KNM, SLM, and WSM-EP-DP can be updated here. I'll merge your suggestions into our internal repo as well.

**Comment by kshiprab** [comment]
Intel folks should be responsible for updating Version and DatePublished in headers when merging the pull request since we are the data owners of event files.

**Comment by diederikdehaas** [comment]
IOW: The changes that I'm requested to make is this?
> Should I remove the file(s) maintained internally and those that are actively released from the DB from this PR?

**Comment by diederikdehaas** [comment]
I just pushed the requested changes (assuming my last message was correct).

No longer fixed in this PR:
```
occurance -> occurrence
occured -> occurred
onthe -> on the
satisifed -> satisfied
```

File: Atom_TMA.csv
Comment (diederikdehaas) [reply Atom_TMA.csv]: Both the TMA files have been removed from this PR as well as the files related to Broadwell and Cascade Lake.
I guess I need to resolve all conversations, but all the others will be without a comment like this.

File: JKT/events/Jaketown_uncore.json
Comment (diederikdehaas) [reply JKT/events/Jaketown_uncore.json:808]: As @kshiprab has an intel.com email address on their GH profile, I assumed that they spoke on behalf of this repo maintainers [here](https://github.com/intel/perfmon/pull/44#issuecomment-1400906404), so I did not increase the version in the header. I'll leave that up to you.

**Comment by diederikdehaas** [comment]
I resolved all the conversations, but it still lists *Changes requested*. I don't know if and if so what else I need to do.
If it's something I need to do, please point out what and where.

**Comment by edwarddavidbaker** [comment]
Thanks for updating the PR @diederikdehaas and for trimming the changes to JKT, KNL, KNM, and WSM-EP-DP. I'll take a look today.

Do you think you could grant me access to your branch? I'd like to add a few more spelling fixes and also bump versions/dates in the header field. Otherwise I can submit a follow-up PR with the remaining updates.

As an example JKT has additional spelling fixes scheduled.
```
perfmon/JKT/events$ git checkout diederik-fork/spelling-fixes
perfmon/JKT/events$ spellcheck-helper.sh | grep == | cut -d':' -f3 | sort | uniq -c | sort -n
      1  Accumultes ==> Accumulates
      1  conjuection ==> conjunction
      1  credts ==> credits
      1  erros ==> errors
      1  happenning ==> happening
      2  MULsand ==> muls and
      2  inavailability ==> unavailability
      2  noncachaeble ==> non-cacheable, uncacheable, noncacheable
      2  oustandings ==> outstandings, outstanding
      3  acquried ==> acquired
      3  threshholding ==> thresholding
      9  cachable ==> cacheable
     11  Sequenser ==> Sequencer
     13  trasmitted ==> transmitted
```

**Comment by edwarddavidbaker** [comment]
Thanks for granting access @diederikdehaas . I'll try to work on the remaining spelling issues for these platforms.

**Comment by edwarddavidbaker** [comment]
Sorry about the delays, I added a few more commits to your branch. The branch was rebased due to changes in `mapfile.csv`. Running a few final checks and will then merge.

**Review by edwarddavidbaker — APPROVED** [review]
LGTM.

**Comment by diederikdehaas** [comment]
Just looked at the latest merge for 6.4-rc1 and what I do I see? ``occurance`` :-/

**Comment by edwarddavidbaker** [comment]
Thanks for checking @diederikdehaas. I currently see instances of `occurance` in TMA related metric files in this repository. We will include them in our spelling checks.

```
$ grep -rn occurance
E-core_TMA_Metrics.csv:64:Info.Inst_Mix,IpBranch,,,,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,,,Instructions per Branch (lower number means higher occurance rate), ,
E-core_TMA_Metrics.csv:65:Info.Inst_Mix,IpCall,,,,INST_RETIRED.ANY / BR_INST_RETIRED.CALL,,,Instruction per (near) call (lower number means higher occurance rate), ,
Atom_TMA.csv:50:Info.Inst_Mix,IpBranch,,,,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,,,Instructions per Branch (lower number means higher occurance rate), ,
Atom_TMA.csv:51:Info.Inst_Mix,IpCall,,,,INST_RETIRED.ANY / BR_INST_RETIRED.CALL,INST_RETIRED.ANY / BR_INST_RETIRED.CALL,,,Instruction per (near) call (lower number means higher occurance rate), ,
```

**Comment by diederikdehaas** [comment]
Ideally you'd link to ``scripts/spelling.txt`` in Linus' ``master`` branch, but copying from there would also work.

**Pull request merged**
